pkg/component/dependencywatchdog/access.go

```diff
@@ -37,10 +41,17 @@ const (
 	DefaultProbeInterval = 30 * time.Second
 	// DefaultWatchDuration is the default value of the total duration for which a DWD Weeder watches for any dependant Pod to transition to CrashLoopBackoff after the target service has recovered.
 	DefaultWatchDuration = 5 * time.Minute
+	// KubeConfigSecretName is the name of the kubecfg secret with internal DNS for external access.
+	KubeConfigSecretName = gardenerutils.SecretNamePrefixShootAccess + "dependency-watchdog-probe"
+	// managedResourceName is the name of the managed resource created for DWD.
+	managedResourceName = "shoot-core-dependency-watchdog"
+
 	// ExternalProbeSecretName is the name of the kubecfg secret with internal DNS for external access.
 	ExternalProbeSecretName = gardenerutils.SecretNamePrefixShootAccess + "dependency-watchdog-external-probe"
 	// InternalProbeSecretName is the name of the kubecfg secret with cluster IP access.
 	InternalProbeSecretName = gardenerutils.SecretNamePrefixShootAccess + "dependency-watchdog-internal-probe"
+	// TempGRMConfigMapName is the name is the configMap that is temporarily deployed for DWD prober
+	TempGRMConfigMapName = "gardener-resource-manager-dwd"
```

Comment:
It seems this name is only used in `app.go`. Remove this constant from this package.